counterparty-core/counterpartycore/lib/api/compose.py

```diff
@@ -630,8 +630,8 @@ def compose_utxo(
     db,
     source: str,
     destination: str,
-    asset: str,
-    quantity: int,
+    asset: str = None,
+    quantity: int = None,
```

Comment:
The unpack function in utxo.py still unpacks the message with an asset and a quantity, so i could presumably compose a detach manually that contains these fields and they will be unpacked and bubbled up?  is that what is desired here?  

https://github.com/CounterpartyXCP/counterparty-core/blob/6904df8242e3d0cd79b5e07fc842587f3bf1bd99/counterparty-core/counterpartycore/lib/messages/utxo.py#L147

Reply:
yeah right!
for now the validate and parse function just ignore them. I hesitated to force them to None but I let like this to avoid another `if util.enabled("spend_utxo_to_detach")`.. but given your comment seems confusing :) I will update this.

(the root problem is that initially attach and detach had the same signature.. not it would be cleaner to have two functions but don't want to change too much code :-/)

Reply:
see https://github.com/CounterpartyXCP/counterparty-core/pull/2626/commits/b9b702782e333de96a4ccc43493c9d2f1efd9405

Reply:
so we have 2 clean contracts now :)